addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -485,6 +485,9 @@ export class WysiwygAdapterComponent extends Wysiwyg {
         // snippets in user databases.
         let $extraEditableZones = $editableSavableZones.find('.s_company_team .o_not_editable *')
             .filter((i, el) => isMediaElement(el) || el.tagName === 'IMG');
+        // Same as above for social media icons.
+        $extraEditableZones = $extraEditableZones.add($editableSavableZones
+            .find('.s_social_media a > i'));
```

Comment:
@qsm-odoo I understand it's not the priority right now but did you create a task so that later we can remove these pieces of code for the existing blocks (with an upgrade script I suppose). Alternatively, I can create one (low priority)